app/src/main/java/com/teamltt/carcare/activity/HomeActivity.java

```diff
@@ -193,4 +188,36 @@ public void toggleLogging(MenuItem item) {
         }
 
     }
+
+    public void displayStaticData() {
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
+        String syncConnPref = preferences.getString(SettingsActivity.KEY_PREF_SYNC_CONN, "");
+
+        TextView tvStaticDataName = ((TextView) findViewById(R.id.tvStaticDataName));
```

Comment:
Update this R.id reference here when you change the xml to underscore_case. The variable should remain camelCase however.